carbonserver/tests/api/test_api.py

```diff
@@ -0,0 +1,109 @@
+import random
+
+import requests
+
+URL = "http://localhost:8008"
+experiment_id = project_id = user_id = api_key = org_id = team_id = email = None
```

Comment:
Proposal : execute this script from tox.ini [integration] env, with the injection of following parameters : 
- DB URL, which might be a shared dev/QA db hosted on clever
- ids, to let us clean the db after the test execution to make it idempotent
- Authorisations conditions, which might be injected later in this script, or might not, as we could separate it in a second integration test.
- Complex test cases. Test data can be handled in a separate file (using pytest fixture import functions) if it becomes difficult to read in this script.

Reply:
The script is run in Github Action with command `tox -e with_api_server`. DATABASE_URL is set by Github Action and CODECARBON_API_URL is set by Tox.

For "ids" we can't force the ids through the API.

When we have a delete entry we could delete all entries we add.